screwdriver/screwdriver.go

```diff
@@ -270,6 +271,10 @@ func (a api) put(url *url.URL, bodyType string, payload io.Reader) ([]byte, erro
 	return a.write(url, "PUT", bodyType, payload)
 }
 
+func (a api) GetAPIURL() string {
+	return a.baseURL
```

Comment:
I feel like the `API URL` should just be the hostname. The namespace `v4` ties more to the endpoint. So if we move that endpoint to somewhere else, we just need to change the code in `template-main` instead of `launcher`. It makes more sense since the `template-main` interacts with the endpoint.